### ISSUE TITLE
Current project architecture 'ARM' is incompatible with the remote system architecture

### ISSUE THREAD

**Issue by nadavb** [Microsoft/VSLinux#285]
I am trying to use Visual Studio 2015, as an IDE for DragonBoard 410c (arm processor, aarch64 architecture).

When I follow the instructions here: https://blogs.msdn.microsoft.com/vcblog/2016/03/30/visual-c-for-linux-development/

And choose the project to be ARM, I get:

C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V140\Application Type\Linux\1.0\Linux.targets(51,5): error : Current project architecture 'ARM' is incompatible with the remote system architecture '' ('Unknown'). Please switch the project architecture to '' in Configuration Manager.

**Comment by stanthomas** [comment]
It's 64-bit. See #110 .

**Comment by itodirel** [comment]
regarding unknown, can you tell us what does uname -m return?

**Comment by nadavb** [comment]
aarch64

**Comment by itodirel** [comment]
Ah I see, we made a fix for that, and are working on adding ARM64 support, in addition to a way of disabling the architecture check.

**Comment by nadavb** [comment]
Okay. In the meanwhile, I switched to Eclipse and continuing the work there.

**Comment by itodirel** [comment]
The architecture checks will be available in 15.8 Preview 2; for the ARM64 support we're working with legal and we'll update you if it can be in the same release or later

**Comment by smlehbleh** [comment]
I was also facing this issue with a 64bit ARM cpu (NVIDIA Jetson TX1 in this instance)
I installed 15.8 Preview 2 and I can confirm that disabling the architecture check removes the "'ARM' is incompatible with the remote system architecture" issue and allows the project to build. I am, however, unable to debug the program. Visual Studio successfully remote runs the application but none of the breakpoints seem to work. Is this a known issue currently? Or have I just got something not setup right?
